processing/src/main/java/org/apache/druid/segment/NestedDataColumnMerger.java

```diff
@@ -93,6 +93,9 @@ public void writeMergedValueDictionary(List<IndexableAdapter> adapters) throws I
         final IndexableAdapter.NestedColumnMergable mergable = closer.register(
             adapter.getNestedColumnMergeables(name)
         );
+        if (mergable == null) {
```

Comment:
Was this existing bug, or does this change somehow allow `mergable` to be nullable now?

Reply:
no, its fine, was just curious.